seedDatabase/query_PostgreSQL.sql

```diff
@@ -0,0 +1,73 @@
+DROP TABLE courses;
+DROP TABLE sections;
+DROP TABLE [elements];
+
+CREATE TABLE courses (
+	course_id SERIAL,
+	id VARCHAR PRIMARY KEY,
```

Comment:
interesting that you used both of these. I ended up setting my courseId as the primary key and serial